packages/accordion/src/Panel.svelte

```diff
@@ -94,7 +94,7 @@
               }
 
               // Assign only when the panel is fully opened.
-              opened = true;
+              opened = open;
 
               dispatch(getElement(), 'SMUIAccordionPanel:opened', { accessor });
```

Comment:
This `opened` event might still fire _after_ any `closed` event though ; wondering if it should simply not be sent or kept as is...